cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -26,10 +26,11 @@ class RecipeLoader(object):
     # TODO: add all default values here
     default_values = default_recipe_values.copy()
 
-    def __init__(self, input_file_path, save_converted_recipe=False):
+    def __init__(self, input_file_path, db_handler=None, save_converted_recipe=False):
```

Comment:
Should probably not be sending a db_handler here? Maybe send it to auto pack. Should also be optional and default to None so when we run our tests and when we run a recipe locally it doesn't have any firebase stuff to do